lib/presenters/artefact_presenter.rb

```diff
@@ -95,6 +95,10 @@ def present
       }
     end
 
+    if downtime = Downtime.for(@artefact)
+      presented['downtime_message'] = downtime.message if downtime.publicise?
```

Comment:
This should really be in the `details` hash, and would probably make sense to model like the other elements above (line 76-89)